workflow-service/src/main/java/com/redhat/parodos/workflow/definition/dto/WorkDefinitionResponseDTO.java

```diff
@@ -61,12 +63,15 @@ public class WorkDefinitionResponseDTO {
 
 	private String author;
 
-	private List<WorkDefinitionResponseDTO> works;
+	private Set<WorkDefinitionResponseDTO> works;
```

Comment:
correct me if I am wrong, but unlike List, Java Set is NOT an ordered collection, it's elements does NOT have a particular order.   this will affect the tasks sequence in workflow topology graph

Reply:
good catch I did not think of it (or was aware of it as it is) but keeping the order IS important, even for testing purposes